sdk/translation/azure-ai-translation-text/tsp-location.yaml

```diff
@@ -1,3 +1,3 @@
 directory: specification/translation/Azure.AI.TextTranslation
-commit: 7b097c7e91a72a8930b94282274158b71e604695
-repo: Azure/azure-rest-api-specs
+commit: 3af1f0b4582e775d1dfefe578781671fc599296f
+repo: MikeyMCZ/azure-rest-api-specs
```

Comment:
Is it generated from personal repo? According to rule, SDK must be generated from public repo https://github.com/Azure/azure-rest-api-specs

Reply:
I was waiting for PR to complete. This is now fixed.